Makefile

```diff
@@ -35,7 +35,7 @@ install-pre-commit:
 install-onnx-test:
 	pip install -U pip
 	pip install -e .
-	pip install "git+https://github.com/openvinotoolkit/open_model_zoo.git@37f60eb#egg=accuracy_checker&subdirectory=tools/accuracy_checker"
+	pip install "git+https://github.com/AlexanderDokuchaev/open_model_zoo.git@94598ef#egg=accuracy_checker&subdirectory=tools/accuracy_checker"
```

Comment:
Please specify the PR with these changes in Model Zoo and I will discuss if it can be merged instead.

Reply:
PR was merged, changed back to openvinotoolkit/open_model_zoo